cms/templates/js/active-video-upload.underscore

```diff
@@ -1,3 +1,8 @@
 <h4 class="video-detail-name"><%- fileName %></h4>
 <progress class="video-detail-progress" value="<%= progress %>"></progress>
-<p class="video-detail-status"><%- gettext(status) %></p>
+<p class="video-detail-status">
+  <%- gettext(
```

Comment:
I'm wondering if it would be better to fix this in the library,r ather than making our code conform.